produce/spec/manager_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe Produce do
+  before do
+    stub_produce
+  end
+  describe "Manager" do
+    it "should auto convert string hash keys to symbol keys" do
+      Produce.config = {
+        username: "helmut@januschka.com",
+        enabled_features: { "data_protection" => "complete" },
+        skip_itc: true
+      }
+      instance = Produce::DeveloperCenter.new
+      features = instance.enabled_features
+      expect(features["dataProtection"].value).to eq("complete")
+    end
+
+    it "accepts symbol'd hash" do
+      Produce.config = {
+        username: "helmut@januschka.com",
+        enabled_features: { data_protection: "complete" },
+        skip_itc: true
+      }
```

Comment:
While this works, it's a little sketchy since the real type of this object should be `Configuration`. It's more common to set this up like:

```ruby
Produce.config = FastlaneCore::Configuration.create(Produce::Options.available_options, {
  username: "helmut@januschka.com",
  enabled_features: { data_protection: "complete" },
  skip_itc: true
})
```